cmd/kube-apiserver/app/server.go

```diff
@@ -294,6 +296,45 @@ func generateStorageVersionMap(legacyVersion string, storageVersions string) map
 	return storageVersionMap
 }
 
+// parse the value of --etcd-servers-overrides and update given storageDestinations.
+func updateEtcdOverrides(overrides []string, storageVersions map[string]string, etcdPathPrefix string, storageDestinations *master.StorageDestinations) {
```

Comment:
My only request is a unit test for this function-- I feel like complicated command line flag parsing should be reasonably well tested.